apps/nextra/pages/zh/build/cli/install-cli/install-cli-linux.mdx

```diff
@@ -0,0 +1,80 @@
+---
+
+title: "在 Linux 上安装 Aptos CLI"
+
```

Comment:
Nit: rm

Reply:
Can we remove all of the extra spaces in the frontmatter titles?

Reply:
> Can we remove all of the extra spaces in the frontmatter titles?

This issue has been fixed in bulk.